go.mod

```diff
@@ -75,16 +75,16 @@ require (
 	go.uber.org/atomic v1.11.0
 	go.uber.org/multierr v1.11.0
 	go.uber.org/zap v1.27.0
-	go.viam.com/api v0.1.388
+	go.viam.com/api v0.1.396
 	go.viam.com/test v1.2.4
 	go.viam.com/utils v0.1.130
 	goji.io v2.0.2+incompatible
 	golang.org/x/image v0.19.0
 	golang.org/x/mobile v0.0.0-20240112133503-c713f31d574b
-	golang.org/x/sync v0.8.0
+	golang.org/x/sync v0.10.0
 	golang.org/x/sys v0.29.0
-	golang.org/x/term v0.25.0
-	golang.org/x/text v0.19.0
+	golang.org/x/term v0.27.0
+	golang.org/x/text v0.21.0
```

Comment:
curious do you know why these got bumped?

Reply:
I believe it's because of https://github.com/viamrobotics/api/pull/625.

Reply:
lol that's me